### PR TITLE
Task-51625: Avoid displaying file name as a link after applying sanitized-html directive and decode encoded file names.

### PR DESCRIPTION
**Pull request by mdiha**

### CODE REVIEW

File: documents-webapp/src/main/webapp/skin/less/documents.less [hunk 1]
```diff
@@ -9,6 +9,11 @@
             border-bottom: none!important;
             .attachment {
               span {
+                a {
+                  color: @textColor!important;
+                  text-decoration-color: var(--allPagesGreyColorLighten1, #a8b3c5);
```
Comment (SaraBoutej) [inline documents-webapp/src/main/webapp/skin/less/documents.less:14]: text-decoration-color: @greyColorLighten1;

Comment (SaraBoutej) [inline documents-webapp/src/main/webapp/skin/less/documents.less]: Please use Color variable name 
text-decoration-color: @greyColorLighten1;

Comment (mdiha) [reply documents-webapp/src/main/webapp/skin/less/documents.less]: Done thx

**Review by SaraBoutej — APPROVED** [review]
LGTM

**Pull request merged**